src/nidcpower/metadata/functions_addon.py

```diff
@@ -147,6 +148,82 @@
+        ],
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession',
+                'documentation': {
+                    'description': 'Identifies a particular instrument session. **vi** is obtained from the niDCPower_InitializeWithChannels function.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'channelName',
+                'type': 'ViChar[]',
+                'documentation': {
+                    'description': '''
+Specifies the output channel(s) to which this configuration value
+applies. Specify multiple channels by using a channel list or a channel
+range. A channel list is a comma (,) separated sequence of channel names
+(for example, 0,2 specifies channels 0 and 2). A channel range is a
+lower bound channel followed by a hyphen (-) or colon (:) followed by an
+upper bound channel (for example, 0-2 specifies channels 0, 1, and 2).
+In the Running state, multiple output channel configurations are
+performed sequentially based on the order specified in this parameter.
+''',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'Count',
+                'type': 'ViInt32',
+                'documentation': {
+                    'description': 'Specifies the number of measurements to fetch.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'Timeout',
+                'type': 'ViReal64',
+                'documentation': {
+                    'description': 'Specifies the maximum time allowed for this function to complete, in seconds. If the function does not complete within this time interval, NI-DCPower returns an error.',
+                    'note': 'When setting the timeout interval, ensure you take into account any triggers so that the timeout interval is long enough for your application.',
+                },
+            },
+            {
+                'direction': 'out',
+                'name': 'measurements',
+                'type': 'ViReal64[]',
+                'documentation': {
+                    'description': 'Returns an array of named tuples.',
+                },
+            },
+        ],
+        'documentation': {
+            'description': '''
+Returns an array of voltage measurements, an array of current
+measurements, and an array of compliance measurements that were
+previously taken and are stored in the NI-DCPower buffer. This function
+should not be used when the NIDCPOWER_ATTR_MEASURE_WHEN attribute is
+set to NIDCPOWER_VAL_ON_DEMAND. You must first call
+niDCPower_Initiate before calling this function.
+
+Refer to the `Acquiring
+Measurements <REPLACE_DRIVER_SPECIFIC_URL_1(acquiringmeasurements)>`__
```

Comment:
Since this is hand-maintained, don't put `<REPLACE_DRIVER_SPECIFIC_URL_1(acquiringmeasurements)>` in there.

Reply:
Paragraph removed